src/dms/EdFi.DataManagementService.sln

```diff
@@ -48,6 +48,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.DataManagementService.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.DataManagementService.OpenApiGenerator.Tests.Integration", "clis\EdFi.DataManagementService.OpenApiGenerator.Tests.Unit\EdFi.DataManagementService.OpenApiGenerator.Tests.Integration.csproj", "{5DD356BC-1CAA-487A-8467-AB85CF6428F2}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EdFi.DataManagementService.Downloader", "clis\EdFi.DataManagementService.Downloader\EdFi.DataManagementService.Downloader.csproj", "{7056F95F-A0D0-4BA4-ACAE-F2A2314E66A4}"
```

Comment:
How about changing this project name to `EdFi.DataManagementService.ApiSchemaDownloader`